src/main/java/com/codecool/dungeoncrawl/logic/actors/Key.java

```diff
@@ -0,0 +1,38 @@
+package com.codecool.dungeoncrawl.logic.actors;
```

Comment:
At first items seems not actor type. If it is true I would put them in their own package